contracts/src/v0.8/dev/vrf/VRFCoordinatorV2Plus.sol

```diff
@@ -459,9 +466,12 @@ contract VRFCoordinatorV2Plus is VRF, SubscriptionAPI {
         s_withdrawableTokens[s_provingKeys[output.keyHash]] += payment;
       }
 
+      bytes memory extraArgs = VRFV2PlusClient._argsToBytes(
```

Comment:
Ditto here re: whether the signature of the extra args is provided.

Reply:
yes this will always have the signature